packages/agoric-cli/src/start.js

```diff
@@ -160,6 +161,12 @@ export default async function startMain(progname, rawArgs, powers, opts) {
     }
   };
 
+  const rmVerbose = async filePath => {
+    log(chalk.green(`removing ${filePath}`));
+    // rm is available on all the unix-likes, so use it for speed.
```

Comment:
TIL that agoric-sdk doesn't support Windows. It's documented in https://docs.agoric.com/guides/getting-started/ but nowhere in the repo itself. Consider adding something to README.md.

Or we could do this cross-platform with rimraf. I suspect it's fast enough.

Reply:
I don't want to add dependencies in this cleanup PR, but have opened #8037 to capture the points.